internal/fortran/fortran.go

```diff
@@ -0,0 +1,422 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package fortran provides utilities to help interface to fortran routines.
+//
+// The utility functions provided in fortran are not intended to be used generally
+// and provide very limited friendliness. They are intended to be used in development
+// of Go ports of FORTRAN routines.
+package fortran
+
+import (
+	"github.com/gonum/blas"
+	"github.com/gonum/blas/blas64"
+)
+
+// TransCharacter returns the FORTRAN character for the given transpose operation.
```

Comment:
I want the code to be completely explicit about what is going on. The code here (more below in comment to answer your broader questions) is intended to be scaffolding for porting FORTRAN routines, not something for general use, thought that may change.